templates/rust/events/src/handler.rs

```diff
@@ -29,7 +34,7 @@ mod tests {
     async fn valid_input() {
         let mut input = Event::default();
         input.set_data("application/json", json!({"name": "bootsy"}));
-        let resp = handle(input).await;
+        let resp = handle(input, web::Data::new(HandlerConfig::default())).await;
```

Comment:
I really wish rust supported optional arguments. :)

Can we put this in a static constant local to the test module and call it `config` or something? Just so it won't look so gross?

Reply:
I ended up adding `config()` function to tests that wraps this. This is the simplest thing I could find as making static traits complicates them for the example and I would like to keep those as simple and reusable as possible. Let me know how it looks.

Reply:
Much better, thanks!